examples/micro/intro/rust/main.rs

```diff
@@ -0,0 +1,94 @@
+// In this sample, we'll use the Rust SDK to create and start a service, which automatically
+// participates in the service discovery, stats management, and ping operations. The service
+// we're going to build exposes two endpoints:
+// * min - returns the minimum value in an array of integers
```

Comment:
Add a line space to (hopefully) render the bullets..

```suggestion

// * min - returns the minimum value in an array of integers
```